src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -58,10 +58,14 @@
 
     public static final String VALID_TASK_NAME_0 = "Assignment 1";
     public static final String VALID_TASK_NAME_1 = "Lab 7";
+    public static final String VALID_TASK_NAME_2 = "Assignment3";
     public static final String VALID_TASK_ID_0 = "T1";
     public static final String VALID_TASK_ID_1 = "T77";
+    public static final String VALID_TASK_ID_2 = "T333";
     public static final String VALID_TASK_DEADLINE_0 = "2021-10-28";
```

Comment:
Did we change the task ddl from DD-MM-YYYY to YYYY-MM-DD?

This is the current error message for adding task

![image](https://user-images.githubusercontent.com/69947227/139666202-6a5df3c4-0f1b-42bc-b5c9-462fce1238f1.png)

Reply:
yes. updated task deadline format to reflect our latest change.